src/app/FakeLib/GitVersionHelper.fs

```diff
@@ -0,0 +1,50 @@
+module Fake.GitVersionHelper
+
+open FSharp.Data
+open Newtonsoft.Json
+open System
+
+type GitVersionProperties = {
+                                Major : int;
+                                Minor : int;
+                                Patch : int;
+                                PreReleaseTag : string;
+                                PreReleaseTagWithDash : string;
+                                PreReleaseLabel : string;
+                                PreReleaseNumber : int;
+                                BuildMetaData : string;
+                                BuildMetaDataPadded : string;
+                                FullBuildMetaData : string;
+                                MajorMinorPatch : string;
+                                SemVer : string;
+                                LegacySemVer : string;
+                                LegacySemVerPadded : string;
+                                AssemblySemVer : string;
+                                FullSemVer : string;
+                                InformationalVersion : string;
+                                BranchName : string;
+                                Sha : string;
+                                NuGetVersionV2 : string;
+                                NuGetVersion : string;
+                                CommitsSinceVersionSource : int;
+                                CommitsSinceVersionSourcePadded : string;
+                                CommitDate : string;
+                            }
+
+
+let GitVersion () =
+    let dirs = [ (environVar "ProgramData")  @@ "chocolatey" @@ "bin" ]
```

Comment:
Gitversion does suggest using chocolatey to install the exe, but I think a more resilient way to find the path to the chocolatey install directory is using the environment variable `ChocolateyInstall`. Then we can add `bin` to that and get the install dir.